app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer_test.go

```diff
@@ -115,6 +116,38 @@ func TestEnsureHttpForwardingRule(t *testing.T) {
 	}
 }
 
+func TestListLoadBalancers(t *testing.T) {
+	lbName := "lb-name"
+	// Should create the forwarding rule as expected.
+	frp := ingresslb.NewFakeLoadBalancers("")
+	namer := utilsnamer.NewNamer("mci1", lbName)
+	frs := NewForwardingRuleSyncer(namer, frp)
+	ipAddr := "1.2.3.4"
+	tpLink := "fakeLink"
+	clusters := []string{"cluster1", "cluster2", "cluster3"}
+
+	if err := frs.EnsureHttpForwardingRule(lbName, ipAddr, tpLink, clusters, false /*force*/); err != nil {
+		t.Fatalf("expected no error in ensuring forwarding rule, actual: %v", err)
+	}
+
+	list, err := frs.ListForwardingRules()
```

Comment:
If we split the code as suggested above, I would love to see more test cases such as:
* No forwarding rule exists
* Forwarding rules exist but none with the right prefix

Reply:
okay, I added those 2 test cases.